src/containers/App.tsx

```diff
@@ -13,33 +18,65 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     backgroundColor: '#F5FCFF',
   },
+  button: {
+    color: '#841584',
+  },
 });
 
 interface IProps {
+  addCounter: () => void;
   decreaseCount: (counterId: number) => void;
   increaseCount: (counterId: number) => void;
+  removeCounter: (counterId: number) => void;
   counters: ICounter[];
 }
 
 const App = (props: IProps) => {
-  const renderCounters = () => {
-    return props.counters.map((counter: ICounter, counterId: number) => {
-      return (
-        <Counter
-          key={counterId}
-          count={counter.count}
-          decreaseCount={() => props.decreaseCount(counterId)}
-          increaseCount={() => props.increaseCount(counterId)}
-        />
-      );
-    });
+  const getKey = (item: ICounter, index: number): string => {
+    return String(index);
+  };
+
+  const renderCounter = ({
+    item,
+    index,
+  }: {
+    item: ICounter;
+    index: number;
+  }) => {
+    return (
+      <Counter
+        key={index}
+        count={item.count}
+        decreaseCount={() => props.decreaseCount(index)}
+        increaseCount={() => props.increaseCount(index)}
+        removeCounter={() => props.removeCounter(index)}
+      />
+    );
   };
 
-  return <View style={styles.container}>{renderCounters()}</View>;
+  return (
+    <View style={styles.container}>
+      <FlatList
```

Comment:
The FlatList does not extend to the edges of the screen making scrolling difficult. (Try scrolling on near edges of the screen.)

Reply:
In fact, now that I test it more, scrolling doesn't seem to be working well at all. I believe this is a flexbox issue.